### PR TITLE
[Eligibility Questionnaire] Progress bar

### PR DESCRIPTION
**Pull request by elenm**
Add the eligibility progress bar.

Also adds some formatting fixes.

progress bar mobile
progress bar desktop

### CODE REVIEW

File: sites/public/pages/eligibility/age.tsx [hunk 1]
```diff
@@ -46,7 +58,7 @@ const EligibilityAge = () => {
             <div className="form-card__pager-row primary">
               <Button
                 styleType={AppearanceStyleType.primary}
-                onClick={() => router.push('/eligibility/disability')}
+                onClick={() => router.push("/eligibility/disability")}
```
Comment (avaleske) [inline sites/public/pages/eligibility/age.tsx:61]: Could we use the Eligibility Sections array to build this URL? it'd be more DRY

**Review by avaleske — APPROVED** [review]
LGTM! Can you add a screenshot to the PR description, and update the changelog?

File: sites/public/pages/eligibility/age.tsx (hunk 1 above)
Comment (elenm) [reply sites/public/pages/eligibility/age.tsx]: Done.

**Comment by elenm** [comment]
I made the requested changes. I also decided to pull the formatting changes into a separate PR (#211), so that it's more clear what edits relate to the progress bar.

**Pull request merged**